bindata/etcd/pod.yaml

```diff
@@ -106,7 +126,7 @@ ${COMPUTED_ENV_VARS}
         name: data-dir
     env:
 ${COMPUTED_ENV_VARS}
-      - name: "ETCD_STATIC_POD_REV"
+      - name: "ETCD_STATIC_POD_VERSION"
```

Comment:
no particular objection, but why the change?

Reply:
to standardize against for example KAS `STATIC_POD_VERSION`. We add ETCD_ extenstion so we get it printed in the etcd logs as signal. But it would not be dependent on the overall PR. Just nice to have and not yet release ENV (introduced in 4.8).